pkg/apis/componentconfig/types.go

```diff
@@ -62,3 +62,213 @@ const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
+
+type KubeletConfiguration struct {
+	// config is the path to the config file or directory of files
+	Config string `json:"config"`
```

Comment:
The ordering of fields in the API is curated. These converted flags are awkward because (a) they are all in one struct and (b) we didn't think of them as an API when they were added. I don't care too much what we do for this initial import of legacy flags into config, but in the future we'll want multiple, semantically grouped, curated config types that aren't sorted alphabetically.